src/utils/utils.js

```diff
@@ -68,3 +68,34 @@ export const getResource = (
 
   return res;
 };
+
+const BYTE_UNITS = {
+  B: 1,
+  KB: 1024 ** 1,
+  MB: 1024 ** 2,
+  GB: 1024 ** 3,
+  TB: 1024 ** 4,
+  PB: 1024 ** 5,
+};
+
+export const formatBytes = (bytes, preferredUnit, fixed = 2) => {
```

Comment:
this could have a better implementation in the future

Reply:
done